string robot_id
string task_type  # "Load" or "Unload"
float32[] source  # [x, y]
float32[] destination  # [x, y]
